### DISCUSSION TITLE
Additional nsswitch providers possible?

### DISCUSSION THREAD

**Discussion by dev-zero** [kanidm/kanidm#2250] [Ideas]
Are there any plans or would it be possible to store data for other nsswitch providers in Kanidm?
In a previous infrastructure based on LDAP (OpenLDAP, then ds389) I made use of storing `sudo` rules in LDAP itself.
Although this is just maintained in `nsswitch.conf` and `sudo` is configured to read it, so this could be implemented differently instead.
But also the recently added `subuid` and `subgid` providers could be very useful and would be a prime example to make it work with just some sane defaults.

**Comment by Firstyear** [comment]
See https://github.com/kanidm/kanidm/issues/240 and https://github.com/kanidm/kanidm/issues/393
